test/interactive.test.js

```diff
@@ -0,0 +1,22 @@
+const suitest = require('suitest-js-api');
+const {assert} = suitest;
+
+describe('My super cool test', () => {
+	before(async() => {
+		// Start test
+		await suitest.startTest('my-super-cool-test');
+	});
+
+	it('should pass', async() => {
+		// Open app
+		await assert.openApp();
+
+		// Start interactive REPL
+		await suitest.interactive();
```

Comment:
Would be great to showcase repeater function + there should be some lines other than `openApp` before and after the `interactive`.